electrum/gui/qml/components/WalletMainView.qml

```diff
@@ -43,7 +43,7 @@ Item {
 
         // Android based send dialog if on android
         var scanner = app.scanDialog.createObject(mainView, {
-            hint: qsTr('Scan an Invoice, an Address, an LNURL-pay, a PSBT or a Channel backup'),
+            hint: qsTr('Scan an Invoice, an Address, an LNURL-pay, or a PSBT. Do NOT scan a Lightning Channel backup.'),
```

Comment:
Should say 

hint: qsTr('Scan an Invoice, Address or a PSBT. Lightning not supported.'),